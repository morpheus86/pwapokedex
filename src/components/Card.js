import React from "react";

const Card = ({ name, idx }) => {
  return (
    <div className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5">
      <img
        alt="pokemon-img"
        src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${idx}.png?size=200x200`}
      />

      <div>
        <h2>{name}</h2>
      </div>
    </div>
  );
};

export default Card;
