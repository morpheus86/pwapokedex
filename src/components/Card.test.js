import { shallow, mount, render } from "enzyme";
import React from "react";
import Card from "./Card";
// import { configure } from "enzyme";
// import Adapter from "enzyme-adapter-react-16";

// configure({
//   adapter: new Adapter()
// });

it("render Card component", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
