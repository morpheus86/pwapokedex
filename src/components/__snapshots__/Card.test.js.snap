// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Card component 1`] = `ShallowWrapper {}`;
